### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by SutirthaChakraborty** [hmartelb/avlit#7]
How did you do the loss function ?

**Comment by hmartelb** [comment]
Please refer to the section "3.3 Implementation details" in our paper. There we state that "The training objective is the negative SI-SDR [30]". You can find such metric in many open-source repositories, although this one only includes the model architecture. Regards.

**Comment by SutirthaChakraborty** [comment]
Just curious, won't you share the training code?
